torch/lib/THD/base/ChannelUtils.cpp

```diff
@@ -181,28 +181,29 @@ int connect(const std::string& address, port_type port, bool wait, int timeout)
       }
 
       socklen_t err_len = sizeof(errno);
-      SYSCHECK(getsockopt(socket, SOL_SOCKET, SO_ERROR, &errno, &err_len));
+      ::getsockopt(socket, SOL_SOCKET, SO_ERROR, &errno, &err_len);
+      /* `errno` is set when:
+       *   1. `getsockopt` has failed
+       *   2. there was awaiting error in socket (we save error on `errno` variable)
```

Comment:
! Sorry, I thought that those points (1., 2.) describe mutually exclusive events. As a result I thought that we've got a value of another error stored in `errno`. 

In that case my "we stored in the errno variable before." doesn't have any sense.

I'd change it to "we saved the error to the `errno` variable"